cmd/util/util.go

```diff
@@ -48,7 +48,27 @@ func GetClusterTier(tierCli string) (string, error) {
 		return "FREE", nil
 	}
 
-	return "", fmt.Errorf("The tier must be either 'Sandbox' or 'Dedicated'")
+	return "", fmt.Errorf("the tier must be either 'Sandbox' or 'Dedicated'")
+}
+
+func ValidateNumFaultsToTolerate(numFaultsToTolerate int32, faultTolerance ybmclient.ClusterFaultTolerance) (bool, error) {
```

Comment:
Do we want to do this? I don't see us validating other things. Is the right thing just to let the API respond with the error?

Reply:
I think it is good to add validations here in the CLI too to prevent the API call similar to how the UI does it. We need to add at other places too if needed.